src/prefect/server/models/workers.py

```diff
@@ -188,7 +188,7 @@ async def update_work_pool(
             [UUID, pendulum.DateTime, orm_models.WorkPool, orm_models.WorkPool],
             Awaitable[None],
         ]
-    ],
+    ] = None,
```

Comment:
this is small but i love it when writing a test uncovers some mild UX edge case like this